express_cut/express_service/urls.py

```diff
@@ -43,6 +43,8 @@
     path('customer', view_role.get_all_customers),
     path('manager', view_role.get_all_managers),
 
+    path('feedback/<int:reservation_id>', views.feedback_views),
```

Comment:
This route should be under reservation. Should be something like reservation/<reservation_id>/feedback

Reply:
Ok, will change.